### PR TITLE
Higher allowed version for module java and lower for zookeeper

### PR DESCRIPTION
**Pull request by hp197**
Tested with java module 2.1.0 and fine.
Downed zookeeper module from < 2.0 to < 1.0 to counter for sudden surprises (v0.7.5 is most current atm).

### CODE REVIEW

File: metadata.json [hunk 1]
```diff
@@ -10,19 +10,19 @@
   "dependencies": [
     {
       "name": "puppet/archive",
-      "version_requirement": ">= 1.0.0 < 2.0.0"
+      "version_requirement": ">= 1.0.0 <= 2.0.0"
```
Comment (bastelfreak) [inline metadata.json]: Can you use '< 3.0.0'?

File: metadata.json [hunk 2]
```diff
@@ -10,19 +10,19 @@
...
     },
     {
       "name": "puppetlabs/java",
-      "version_requirement": ">= 1.4.2 < 2.0.0"
+      "version_requirement": ">= 1.4.2 <= 2.1.0"
```
Comment (bastelfreak) [inline metadata.json]: Can you use '< 3.0.0'?

**Comment by bastelfreak** [comment]
Hi @hp197, thanks for the PR! I added two inline comments.

**Comment by hp197** [comment]
You put a lot of trust in the maintainers of the other modules and on the other side you want to keep maintainability yourself... Cant blame you for putting it on major revisions.

**Comment by bastelfreak** [comment]
Thanks @hp197. We noticed in the past that other modules evolve very fast. We constantly need to adjust the minor versions, otherwise the puppet-module-tool isn't able to resolve the dependencies.

**Pull request merged**